server/aws-lsp-codewhisperer/src/language-server/codeWhispererSecurityScanServer.ts

```diff
@@ -47,21 +47,31 @@ export const SecurityScanServerToken =
             }
             try {
                 if (!credentialsProvider.hasCredentials('bearer')) {
-                    throw new Error('credentialsProvider does not have bearer token credentials')
+                    throw new Error('Credentials provider does not have bearer token credentials')
                 }
+
+                logging.log(`Parameters provided: ${JSON.stringify(params)}`)
+
                 if (!params.arguments || params.arguments.length === 0) {
-                    throw new Error(`Incorrect params provided. Params: ${params}`)
+                    throw new Error(`Error: Invalid data.`)
                 }
                 const [arg] = params.arguments
+
+                logging.log(`Arguments provided: ${JSON.stringify(arg)}`)
```

Comment:
Could this lead to sensitive data being logged or is it just path information?

Reply:
This is just path information.